packages/lib-user/src/components/shared/AllProjects/AllProjects.stories.js

```diff
@@ -0,0 +1,39 @@
+import { PROJECTS } from '../../../../test/mocks/panoptes'
+
+import AllProjects from './AllProjects.js'
```

Comment:
(Will write some unit tests once this shared component is in use for UserHome > All Projects and UserStats > AllProjects)